src/widgets/if_element.rs

```diff
@@ -27,6 +31,7 @@ pub struct IfProps {
 /// | `children`  | ✅        |
 /// | `styles`    | ✅        |
 /// | `on_event`  | ✅        |
+/// | `on_layout` | ✅        |
```

Comment:
I don't think we can do this for the `If` widget since it has no layout (i.e. `RenderCommand::Empty`).